src/librustc_codegen_ironox/function.rs

```diff
@@ -35,7 +41,7 @@ impl IronOxFunction {
                 IronOxFunction {
                     name: name.to_string(),
                     basic_blocks: vec![],
-                    stack_size: 8,
+                    stack_size: 8, // the size of rbp
```

Comment:
total nitpick, but all other comments are terminated with a dot.

Reply:
I removed `stack_size` in my last commit.